doc/proposal/in-progress/07-signing.md

```diff
@@ -0,0 +1,230 @@
+            value
+```
+Exceptions:
+
+1. If resource.Access is emtpy or resource.access.type is None, no digest field is required. Then, only name, version, extraIdentity, type and relation is used for the normalisation.
+
+### Properties **NOT** part of normalisation:
+
+component
+    repositoryContexts
+    labels
+    sources
+    componentReference:
+        labels
+    resource:
+        labels
+        access
+signatures
+
+### Representation
+
+In order to hash a normalised component-descriptor, it has to be transformed into a hashable representation.
+
+Attributes *MUST* be transformed into list of single attributes (key-value pairs) and ordered alphabetically. This list *MUST* be JSON encoded with disabled HTML Escaping. It *MUST NOT* end with a newline character. A hash algorithm of choice can then be applied on the string 
+
+Example (pretty printed for readability): *TODO*: update
+```
+[
+  {
+    "component": [
+      {
+        "componentReferences": [
+          [
+            {
+              "componentName": "compRefNameComponentName"
+            },
+            {
+              "digest": [
+                {
+                  "hashAlgorithm": "sha256"
+                },
+                {
+                  "normalisationAlgorithm": "jsonNormalisation/V1"
+                },
+                {
+                  "value": "00000000000000"
+                }
+              ]
+            },
+            {
+              "extraIdentity": [
+                {
+                  "refKey": "refName"
+                }
+              ]
+            },
+            {
+              "name": "compRefName"
+            },
+            {
+              "version": "v0.0.2compRef"
+            }
+          ]
+        ]
+      },
+      {
+        "name": "CD-Name"
+      },
+      {
+        "resources": [
+          [
+            {
+              "digest": [
+                {
+                  "hashAlgorithm": "sha256"
+                },
+                {
+                  "normalisationAlgorithm": "manifestDigest/V1"
+                },
+                {
+                  "value": "00000000000000"
+                }
+              ]
+            },
+            {
+              "extraIdentity": [
+                {
+                  "key": "value"
+                }
+              ]
+            },
+            {
+              "name": "Resource1"
+            },
+            {
+              "version": "v0.0.3resource"
+            }
+          ]
+        ]
+      },
+      {
+        "version": "v0.0.1"
+      }
+    ]
+  },
+  {
+    "meta": [
+      {
+        "schemaVersion": "v2"
+      }
+    ]
+  }
+]
+
+```
+
+### Digester
+
+As described, resources have a digest field to store the content hash. Different resource types will use a different normalisationAlgorithm: 
+
+ - `ociArtifactDigest/v1`: uses the hash of the manifest of an oci artifact
+ - `genericBlobDigest/v1`: uses the hash of the blob
```

Comment:
As described before, the digest of an oci image is computed by the hash of its manifest and this should be the hash of its blob. Therefore I do not understand the difference between ociArtifactDigest and genericBlobDigest. Could be perhaps also add some links to the spec of both methods?

Reply:
the digest of an oci image is a hash of its manifest and the manifest contains hashes of the layers in a structured form. This is the difference to genericBlobDigest, which simply hashes the bytes of a blob. Taking an oci image as a single blob would e.g. result in a tar archive with all layers, which would result in a different digest.

I agree that we should specify it in more detail here.